MapboxNavigation/InstructionPresenter.swift

```diff
@@ -103,3 +135,17 @@ class ShieldAttachment: NSTextAttachment {
         return CGRect(x: 0, y: font.descender - image.size.height / 2 + mid + 2, width: image.size.width, height: image.size.height).integral
     }
 }
+
+extension CGSize {
+    fileprivate static var greatestFiniteSize = CGSize(width: CGFloat.greatestFiniteMagnitude, height: CGFloat.greatestFiniteMagnitude)
```

Comment:
@bsudekum should we consider an `MB` (or whatever) prefix to specific vars we add to extensions. This can prevent any future variable conflicts or overrides with Apple iOS specific classes, structs, enum etc. It would also help developers identify `MB` specific should we exposed certain variables in the future.

Reply:
Not necessary for private or fileprivate where we don't expose anything to ObjC.